NBitcoin.Tests/AltcoinTests.cs

```diff
@@ -186,14 +227,22 @@ public void CanParseAddress()
 		/// This test check if we can scan RPC capabilities
 		/// </summary>
 		[Fact]
-		public void DoesRPCCapabilitiesWellAdvertised()
+		public async Task DoesRPCCapabilitiesWellAdvertised()
 		{
 			using (var builder = NodeBuilderEx.Create())
 			{
 				var node = builder.CreateNode();
 				builder.StartAll();
-				node.Generate(builder.Network.Consensus.CoinbaseMaturity + 1);
 				var rpc = node.CreateRPCClient();
+				if (new[] {Liquid.Instance.Mainnet, Liquid.Instance.Testnet, Liquid.Instance.Regtest}.Contains(node.Network))
```

Comment:
nop, don't do this. Because all tests depends on `GenerateAsync`, if `DoesRPCCapabilitiesWellAdvertised` is well implemented, it should use `GenerateToAddress` under the hood.